example/android/gradle.properties

```diff
@@ -1,4 +1,3 @@
 org.gradle.jvmargs=-Xmx1536M
-android.enableR8=true
```

Comment:
Why are we disabling the R8? Is it causing some issues? Or is it enabled by default, so there's no need to do it explicitly? :thinking:

Reply:
The `android.enableR8` flag is deprecated with the latest versions of Gradle